_includes/charts/calico/templates/kdd-crds.yaml

```diff
@@ -1,209 +1,8 @@
 {{- if eq .Values.datastore "kubernetes" -}}
-apiVersion: apiextensions.k8s.io/v1beta1
-kind: CustomResourceDefinition
-metadata:
-  name: felixconfigurations.crd.projectcalico.org
```

Comment:
If you remove these CRDs from the `calico` helm chart, it will break the chart and any forked versions of it :eyes: 

I think you want to leave these where they are and duplicate the CRDs into the tigera-operator chart. Duplication is gross, but the alternatives are grosser:
- we could double render the chart by using jekyll includes to pull the CRDs from somewhere
- we could import the CRDs using a helm subchart
- we could remove the calico chart now that we're all-in on operator?

Reply:
It is a helm feature to do `Files.Glob` and `Files.Get`. Isn't helm rendering this file?

Reply:
In other words I thought this file would have the same contents as before (though the resources are probably in a different order).

Reply:
Oh shoot, I misread and thought you moved it from the `calico` chart into another chart. This comment is void :)